packages/systemd-257/systemd-257.spec

```diff
@@ -0,0 +1,799 @@
+%{_cross_bindir}/systemd-sysusers
+%{_cross_bindir}/systemd-tmpfiles
+%{_cross_bindir}/systemd-umount
+%{_cross_bindir}/udevadm
+%{_cross_bindir}/loginctl
+%{_cross_bindir}/varlinkctl
+
+%exclude %{_cross_bindir}/run0
+%exclude %{_cross_libdir}/pam.d/systemd-run0
+
+%{_cross_sbindir}/halt
+%{_cross_sbindir}/init
+%{_cross_sbindir}/poweroff
+%{_cross_sbindir}/reboot
+%{_cross_sbindir}/shutdown
+
+%exclude %{_cross_bindir}/systemd-vpick
+%exclude %{_cross_sbindir}/mount.ddi
+
+%{_cross_libdir}/libsystemd.so.*
+%{_cross_libdir}/libudev.so.*
+
+%dir %{_cross_libdir}/systemd
+%{_cross_libdir}/systemd/libsystemd-core-*.so
+%{_cross_libdir}/systemd/libsystemd-shared-*.so
+%{_cross_libdir}/systemd/systemd
+%{_cross_libdir}/systemd/systemd-boot-check-no-failures
+%{_cross_libdir}/systemd/systemd-cgroups-agent
+%{_cross_libdir}/systemd/systemd-executor
+%{_cross_libdir}/systemd/systemd-fsck
+%{_cross_libdir}/systemd/systemd-growfs
+%{_cross_libdir}/systemd/systemd-journald
+%{_cross_libdir}/systemd/systemd-logind
+%{_cross_libdir}/systemd/systemd-makefs
+%{_cross_libdir}/systemd/systemd-modules-load
+%{_cross_libdir}/systemd/systemd-network-generator
+%{_cross_libdir}/systemd/systemd-pstore
+%{_cross_libdir}/systemd/systemd-random-seed
+%{_cross_libdir}/systemd/systemd-remount-fs
+%{_cross_libdir}/systemd/systemd-shutdown
+%{_cross_libdir}/systemd/systemd-sleep
+%{_cross_libdir}/systemd/systemd-socket-proxyd
+%{_cross_libdir}/systemd/systemd-sysctl
+%{_cross_libdir}/systemd/systemd-sysroot-fstab-check
+%{_cross_libdir}/systemd/systemd-udevd
+%{_cross_libdir}/systemd/profile.d/70-systemd-shell-extra.sh
+
+%exclude %{_cross_libdir}/systemd/repart/*
+%exclude %{_cross_libdir}/systemd/system-generators/systemd-ssh-generator
+%exclude %{_cross_libdir}/systemd/system-generators/systemd-tpm2-generator
+
+%exclude %{_cross_libdir}/systemd/system/tpm2.target
+%exclude %{_cross_libdir}/systemd/systemd-battery-check
+%exclude %{_cross_libdir}/systemd/systemd-ssh-proxy
+
+%exclude %{_cross_libdir}/systemd/system/capsule.slice
+%exclude %{_cross_libdir}/systemd/system/capsule@.service
+%exclude %{_cross_libdir}/systemd/system/sockets.target.wants/systemd-creds.socket
+%exclude %{_cross_libdir}/systemd/system/soft-reboot.target
+%exclude %{_cross_libdir}/systemd/system/ssh-access.target
+%exclude %{_cross_libdir}/systemd/system/sysinit.target.wants/systemd-tmpfiles-setup-dev-early.service
+%exclude %{_cross_libdir}/systemd/system/systemd-creds.socket
+%exclude %{_cross_libdir}/systemd/system/systemd-creds@.service
+%exclude %{_cross_libdir}/systemd/system/systemd-growfs-root.service
+%exclude %{_cross_libdir}/systemd/system/systemd-growfs@.service
+%exclude %{_cross_libdir}/systemd/system/systemd-journald-sync@.service
+%exclude %{_cross_libdir}/systemd/system/systemd-soft-reboot.service
+%exclude %{_cross_libdir}/systemd/system/systemd-tmpfiles-setup-dev-early.service
+%exclude %{_cross_libdir}/systemd/system/systemd-udev-load-credentials.service
```

Comment:
Unit-related includes and excludes should be in the `%{_cross_unitdir}` and `%exclude %{_cross_unitdir}` sections below.

Anything excluded should have a comment saying why it's excluded.